internal/configs/config.go

```diff
@@ -0,0 +1,67 @@
+package configs
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/spf13/viper"
+)
+
+type Config struct {
```

Comment:
I sorted to use a custom function to validate the config.
I will check the validator package doc and revisit this comment later if it can make things better/easier.